macros/Cargo.toml

```diff
@@ -5,6 +5,9 @@ edition = "2021"
 repository = "https://github.com/kyrias/sqlx-conditional-queries"
 license = "MIT OR Apache-2.0"
 
+# REVIEW: Why did you add this crate or rather, why did you add the core crate?
+# REVIEW: From what I've seen, the `core` crate doesn't expose any structs or functions that are of interest outside of the `macros` crate.
+# REVIEW: Wouldn't it be easier to just move all of `core`'s logic to `macros`, since all of its logic is macro related?
```

Comment:
Proc macro crates can only expose macros, making it very hard to test the different parts if all of the core functionality is in the macro crate.  _Currently_ it doesn't matter because all the tests are in-line unit tests, but I'm planning on eventually having a fuller test suite that won't be in-line test modules.